step-release-vis/src/app/components/grid/svg-grid/svg-grid.component.html

```diff
@@ -1,6 +1,5 @@
 <!DOCTYPE html>
```

Comment:
this is now the same as the other template... maybe consider reusing the same template e..g empty.html from a common location.

Also serving the angular page will need the compoenents to be html elements not pages, so we only need the part what is inside the body, not the html and body tags.

Reply:
Is it okay if both files just have `<div id="container"></div>`? It works this way, and to load the template into the component's html we'll probably need more than one line, and it will still be duplicate code fro both components.

Reply:
yeah, but I would actually prefer to have it a unique name in the final version.. also if it is a single line, we can use it from the code, and not have it in a separate file